docs/csharp/programming-guide/strings/index.md

```diff
@@ -1,156 +1,182 @@
+
+For more information about how to create new strings that are based on modifications such as search and replace operations on the original string, see [How to modify string contents](../../how-to/modify-string-contents.md).
+
+## Quoted string literals
+
+*Quoted string literals* are start and end with a single double quote character (`"`) on the same line. Quoted string literals are best suited for strings that fit on a single line and don't include any [escape sequences](#string-escape-sequences). A quoted string literal must embed escape characters, as shown in the following example:
+
+:::code language="csharp" source="./snippets/StringLiterals.cs" id="EscapeSequences":::
+
+## Verbatim string literals
+
+*Verbatim string literals* are more convenient for multi-line strings, strings that contain backslash characters, or embedded double quotes. Verbatim strings preserve new line characters as part of the string text. Use double quotation marks to embed a quotation mark inside a verbatim string. The following example shows some common uses for verbatim strings:
+
+:::code language="csharp" source="./snippets/StringLiterals.cs" id="VerbatimLiterals":::
+
+## Raw string literals
+
+Beginning with C# 11, you can use *raw string literals* to more easily create strings that are multi-line, or use any characters requiring escape sequences. *Raw string literals* remove the need to ever use escape sequences. You can write the string, including whitespace formatting, how you want it to appear in output. A *raw string literal*:
+
+- Starts and ends with a sequence of at least three double quote characters (`"""`). You're allowed more than three consecutive characters to start and end the sequence in order to support string literals that contain three (or more) repeated quote characters.
+- Single line raw string literals require the opening and closing quote characters on the same line.
+- Multi-line raw string literals require both opening and closing quote characters on their own line.
+- In multi-line raw string literals, any whitespace to the left of the closing quotes are removed.
+
+The following examples demonstrate these rules:
+
+:::code language="csharp" source="./snippets/StringLiterals.cs" id="RawStringLiteralSyntax":::
+
+The following examples demonstrate the compiler errors reported based on these rules:
+
+:::code language="csharp" source="./snippets/StringLiterals.cs" id="ErrorExamples":::
+
+You should consider raw string literals when you're generating text that includes characters that require [escape sequences](#string-escape-sequences) when using quoted string literals or verbatim string literals. Raw string literals will be easier for you and others to read because it will more closely resemble the output text. For example, consider the following code that includes a string of formatted JSON:
+
+:::code language="csharp" source="./snippets/StringLiterals.cs" id="JSONString":::
```

Comment:
@tdykstra Here's where I included a raw string literal version of the JSON example.